### PR TITLE
(apache-httpd) Fixed compatibility with Powershell v2

### PR DESCRIPTION
**Pull request by RedBaron2**
Fixed the non-Powershell v2 compatibility comlet of Get-NetTCPConnection
@majkinetor @AdmiringWorm @gep13 

## Description
This solution uses netstat for retrieval of the information requested using the windows 8 or newer system comlet of Get-NetTCPConnection. Has been tested on a windows 7 machine.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [X] Issue #828 
  - [ ] Issue 2 Link
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/apache-httpd/tools/helpers.ps1 [hunk 1]
```diff
@@ -4,8 +4,8 @@ function Assert-TcpPortIsOpen {
         [Parameter(Position = 0, Mandatory, ValueFromPipeline)][ValidateNotNullOrEmpty()][int] $portNumber
     )
 
-    $process = Get-NetTCPConnection -State Listen -LocalPort $portNumber -ErrorAction SilentlyContinue | `
-        Select-Object -First 1 -ExpandProperty OwningProcess | `
+    $process = ( netstat -aon | foreach { if ( $_ -match ":$portNumber" ) { $_ } } )-split " " | `
```
Comment (majkinetor) [inline automatic/apache-httpd/tools/helpers.ps1]: `netstat -aon | ? {$_ -match ... }` or `(netstat -aon) -match ...`

Comment (RedBaron2) [reply automatic/apache-httpd/tools/helpers.ps1]: Sounds like a good suggestion. I'll change it now

**Comment by RedBaron2** [comment]
@majkinetor 
Changes made and Appveyor agrees 👍

File: automatic/apache-httpd/tools/helpers.ps1 [hunk 2]
```diff
@@ -4,7 +4,7 @@ function Assert-TcpPortIsOpen {
         [Parameter(Position = 0, Mandatory, ValueFromPipeline)][ValidateNotNullOrEmpty()][int] $portNumber
     )
 
-    $process = ( netstat -aon | foreach { if ( $_ -match ":$portNumber" ) { $_ } } )-split " " | `
+    $process = (( netstat -aon ) -match ":$portNumber" ) -split " " | `
```
Comment (majkinetor) [inline automatic/apache-httpd/tools/helpers.ps1]: match is not good, please fix regex. This one will return invalid result, for example you searching for `:80` may return `:8080`.

Comment (RedBaron2) [reply automatic/apache-httpd/tools/helpers.ps1]: fixed the match

**Comment by RedBaron2** [comment]
@majkinetor 
I made the match so it will old match if that number is found not inside a port

File: automatic/apache-httpd/tools/helpers.ps1 [hunk 3]
```diff
@@ -1,10 +1,53 @@
+
```
Comment (majkinetor) [inline automatic/apache-httpd/tools/helpers.ps1]: Fix code style and identation and remove extra empty lines, to be like the rest of the code.

**Comment by RedBaron2** [comment]
@majkinetor 
Fixed code style and indentation

**Pull request merged**